src/finance_quote/base.py

```diff
@@ -19,18 +19,19 @@ class Symbol:
 class Quote:
     """Class to represent a quote (price, ...)"""
     def __init__(self):
-        self.datetime: datetime = None
-        self.namespace: str = None
-        self.symbol: str = None
-        self.value: Decimal = Decimal(0)
-        self.currency: str = None
+        self.datetime = None # : datetime
```

Comment:
this kind of assignment should be done at class level not in the `__init__` except if the latter accepts arguments to initialise these fields

Reply:
I tried that before but there were weird consequences as those variables were static and shared among all instances of the class. Then I read that this is the way to declare and initialize instance-level variables. Perhaps I'm wrong.

Reply:
The current code is there just to declare the fields (for documentation & type hinting), correct ?
Strictly speaking, it is not needed in Python.
If you declared it a class level, as they are simple types (not containers like lists, dicts, etc), they will be overwritten by instance levrl attributes whenever you do a `self.field= ...`

Reply:
Yes, correct. We can try that. I just have a bad feeling from doing it the last time. This way just seems like a safer option by default.